code/game/objects/items/melee/f13onehanded.dm

```diff
@@ -26,29 +26,6 @@
 // SWORDS //
 ////////////		-block, 34-39 damage
 
-
```

Comment:
What's with removal of this?
Could at least make it a generic admin spawn item, if the intent is to deprecate fluff items like this.

Reply:
We can always add generic admin spawn items that are thematic to Fallout in the grand scope of things. Not... Whatever that katana was.